README.md

```diff
@@ -1,7 +1,13 @@
 # prplMesh
 [![Build Status](https://travis-ci.com/prplfoundation/prplMesh.svg?branch=master)](https://travis-ci.org/prplfoundation/prplMesh)
 
-TODO describe this project
+prplMesh is an open-source, carrier-grade and certifiable implementation of the WiFi Alliance’s Multi-AP specification.
+
+The result will be source-code covering both the agent and the controller part of the Multi-AP specification. However, it is scoped as a reference implementation and will leave ample room for differentiation, for example for proprietary IP algorithms making intelligent decisions for the whole Multi-AP network.
```

Comment:
As noted in [the coding style](https://github.com/prplfoundation/prplMesh/blob/4a86f5378d8bf77148dcfa1da474a47bd31416c1/CODINGSTYLE.md#documentation), markdown files should have one sentence per line. This gives the best diffs when changing some words within a paragraph.